index.bs

```diff
@@ -1411,11 +1416,19 @@ To <dfn>parse a filter pair</dfn> given a [=map=] |map|:
 <h3 id="debug-keys">Cookie-based debugging</h3>
 
 To <dfn>check if cookie-based debugging is allowed</dfn> given a
-[=suitable origin=] |reportingOrigin|:
+[=suitable origin=] |reportingOrigin| and a [=site=] |contextSite|:
 
+1. [=Assert=]: |contextSite| is not the [=opaque origin=].
 1. Let |domain| be the
     <a href="https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis#name-canonicalized-host-names">canonicalized domain name</a>
     of |reportingOrigin|'s [=origin/host=].
+1. Let |contextDomain| be the
+    <a href="https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis#name-canonicalized-host-names">canonicalized domain name</a>
+    of |contextSite|'s [=scheme and host/host=].
+1. If the User Agent's <a href="https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis#name-cookie-policy">cookie policy</a>
+    or <a href="https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis#name-user controls">user controls</a>
+    do not allow cookie access for |domain| on |contextDomain| within a <a href="https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis#name-third-party-cookies">third-party context</a>,
+    return <strong>blocked</strong>.
```

Comment:
Could you add some context for this change  to the PR description? It doesn't seem strictly related to the subject of the PR, since it also affects source debug-cookie checks separately from trigger-verbose-debug-report ones.

Reply:
Thanks, updated